qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -228,6 +228,15 @@ def get_session(self, client_id, session_ls=None):
     def _session_by_id(self, session_ls):
         return dict([(s['id'], s) for s in session_ls])
 
+    def wait_until_evicted(self, client_id, timeout=30):
+        def helper():
+            ls = self._session_list()
+            for s in ls:
+                if s['id'] == client_id:
+                    return False
+            return True
+        self.wait_until_true(lambda: helper(), timeout)
```

Comment:
```suggestion
        self.wait_until_true(helper, timeout)
```

also, we could use a better name than `helper`. like `is_client_evicted`.

Reply:
or, 
```py
def client_evicted():
  return all(s['id] != client_id for s in self._session_list())
self.wait_until_true(client_evicted, timeout)
```

Reply:
> return all(s['id] != client_id for s in self._session_list())

Personally I hate this Pythonism so I try to avoid it. I fixed your other suggestions. Thanks @tchaikov